scripts/jupyter-debug.sh

```diff
@@ -77,5 +77,5 @@ echo
 echo '========================'
 echo 'Content of ~/jupyter.log'
```

Comment:
For completeness, the echoed message must be changed to `~/.jupyter.log` as well